### PR TITLE
Remove experimental from manifest subcommand

### PR DESCRIPTION
**Pull request by ostromart**
manifest apply subcommand is no longer experimental in 1.4.

### CODE REVIEW

File: istioctl/cmd/root.go [hunk 1]
```diff
@@ -148,7 +148,7 @@ debug and diagnose their Istio mesh.
 
 	manifestCmd := mesh.ManifestCmd()
 	hideInheritedFlags(manifestCmd, "namespace", "istioNamespace")
-	experimentalCmd.AddCommand(manifestCmd)
+	rootCmd.AddCommand(manifestCmd)
 
 	profileCmd := mesh.ProfileCmd()
 	hideInheritedFlags(profileCmd, "namespace", "istioNamespace")
```
Comment (elfinhe) [inline istioctl/cmd/root.go]: also remove `profile` from experimental?

Comment (ostromart) [reply istioctl/cmd/root.go]: Great point, thanks Tao.

**Comment by ostromart** [comment]
why is this not merging @howardjohn ?

**Comment by howardjohn** [comment]
> Waiting on code owner review from @istio/wg-user-experience-maintainers.

**Pull request merged**